### ISSUE TITLE
Get payment history information wired through to the fines controller

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/mylibrary#137]
<img width="383" alt="Screen Shot 2019-07-11 at 19 42 49" src="https://user-images.githubusercontent.com/111218/61098506-16c7ba00-a414-11e9-8fe0-e1c0289a8de5.png">

Sort options:
<img width="133" alt="Screen Shot 2019-07-16 at 07 42 50" src="https://user-images.githubusercontent.com/111218/61304017-550b0380-a79d-11e9-90e4-4d9c0a9a0d82.png">

(The accruals section is covered in #140)

Data needed:

amount [Paid, Forgiven, etc]
Date resolved
Title

Reason (possibly more detailed than brief status)
Billed on
Paid|Forgiven|Returned|Removed on: date

**Comment by camillevilla** [comment]
Assigning this to @shelleydoljack and @jgreben while they do some of the pre-requisite work. Feel free to split a ticket out for the UI stuff if other issues become more pressing.

**Comment by jgreben** [comment]
Response from SD about payment history in SymWS:

"Thank you for contacting SirsiDynix.  In the current version the legacy call is the only way to get this information.  A blockHistoryList is planned to be added to the /user/patron information in the next version, once that has been released and you have upgraded you will be able to get this something like:"
```
GET /user/patron/key/{userkey}?includeFields=blockHistoryList{*}
```

For now I made a `SymphonyLegacyClient` service and we can review the implementation of that.
